x-pack/plugins/observability/public/components/app/news_feed/index.tsx

```diff
@@ -79,16 +80,19 @@ const NewsItem = ({ item }: { item: NewsItem }) => {
               </EuiFlexItem>
             </EuiFlexGroup>
           </EuiFlexItem>
-          <EuiFlexItem grow={false}>
-            <img
-              style={{ border: theme.eui.euiBorderThin }}
-              width={48}
-              height={48}
-              alt={item.title}
-              src={item.image_url}
-              className="obsNewsFeed__itemImg"
-            />
-          </EuiFlexItem>
+          {item.image_url?.en && (
+            <EuiFlexItem grow={false}>
+              <img
+                data-test-subj="news_image"
+                style={{ border: theme.eui.euiBorderThin }}
+                width={48}
+                height={48}
+                alt={item.title.en}
+                src={item.image_url.en}
+                className="obsNewsFeed__itemImg"
```

Comment:
what's this class?

Reply:
To include a sass style `@include euiBottomShadowSmall;`